packages/wdio-browserstack-service/src/Percy/PercySDK.ts

```diff
@@ -0,0 +1,46 @@
+import { createRequire } from 'node:module'
+const require = createRequire(import.meta.url)
+
+const tryRequire = function (pkg: string, fallback: any) {
+    try {
+        return require(pkg)
+    } catch {
+        return fallback
+    }
+}
+
+const percySnapshot = tryRequire('@percy/selenium-webdriver', null)
```

Comment:
I am a bit concerned that this asks users to install `selenium-webdriver` as peer dependency. WebdriverIO is a complete different implementation.

I suggest to either create a more general package that doesn't require `selenium-webdriver` as dependency or move the code in here.